WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -852,6 +852,29 @@ private extension ProductFormViewController {
     }
 }
 
+// MARK: Action - Edit Grouped Products (Grouped Products Only)
+//
+private extension ProductFormViewController {
+    func editGroupedProducts() {
+        let viewController = GroupedProductsViewController(product: product) { [weak self] groupedProductIDs in
+            self?.onEditGroupedProductsCompletion(groupedProductIDs: groupedProductIDs)
+        }
+        show(viewController, sender: self)
+    }
+
+    func onEditGroupedProductsCompletion(groupedProductIDs: [Int64]) {
+        defer {
+            navigationController?.popViewController(animated: true)
+        }
+        // TODO-2000: Edit Product M3 analytics
```

Comment:
```suggestion
        // TODO-2509: Edit Product M3 analytics
```

Reply:
updated in: https://github.com/woocommerce/woocommerce-ios/pull/2505/commits/60177e57082e3947d5a4da86f323dbb56e11e810